torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -151,16 +151,34 @@ llvm::Error contractParamIfNeeded(const glow::Handle<T> &handle,
   return llvm::Error::success();
 }
 
-/// Given Node inputs and outputs, check the expected sizes.
+/// Given a handle to a Constant with only one element in it \p handle, puts
+/// that element in \p elem. \returns an error if anything went wrong.
+template <typename T, typename OutT>
+llvm::Error getSingleValueFromConstant(const glow::Handle<T> &handle,
+                                       OutT &elem) {
+  RETURN_ERR_IF_NOT(handle.size() == 1,
+                    glow::strFormat("Handle size must be equal to 1, got %lu.",
+                                    handle.size()));
+  elem = handle.raw(0);
+  return llvm::Error::success();
+}
+
+/// Given Node inputs and outputs, check the expected sizes. Negative size
+/// indicates that the size could be anything so we don't check.
 template <typename T>
-llvm::Error checkInputAndOutputSizes(const T &inputs, size_t inputsSize,
-                                     const T &outputs, size_t outputsSize) {
-  RETURN_ERR_IF_NOT(inputs.size() == inputsSize,
-                    glow::strFormat("Expected number inputs %lu, got %lu.",
-                                    inputsSize, inputs.size()));
-  RETURN_ERR_IF_NOT(outputs.size() == outputsSize,
-                    glow::strFormat("Expected number outputs %lu, got %lu.",
-                                    outputsSize, outputs.size()));
+llvm::Error checkInputAndOutputSizes(const T &inputs, int64_t inputsSize,
```

Comment:
Method can be extended to check not only sizes equality, == but also >=, may be negative size can be treated as >=.